management/tenant_test.go

```diff
@@ -41,6 +41,11 @@ func TestTenantManager(t *testing.T) {
 		Sessions: &TenantSessions{
 			OIDCLogoutPromptEnabled: auth0.Bool(false),
 		},
+		AcrValuesSupported:                   &[]string{"foo", "bar"},
```

Comment:
we need tests for both setting this to null in the api, and omitting it in the API